include/mbgl/shaders/gl/circle.hpp

```diff
@@ -1,5 +1,5 @@
 // Generated code, do not modify this file!
-// Generated on 2023-05-12T17:51:34.322Z by mwilsnd using shaders/generate_shader_code.js
```

Comment:
In order to avoid conflicts it will be good to revert these files, that doesn't have any important changes.

Reply:
I've been avoiding committing those, but it's a bit tedious.  The same basic info is in git-blame ("authors" pane in Xcode), so I would advocate removing the date and username.

Reply:
+1